src/features/home/ui/ChartHeader.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { Meta, StoryObj } from '@storybook/react';
+import ChartHeader from './ChartHeader';
+
+const meta: Meta<typeof ChartHeader> = {
+  title: 'component/home/ChartHeader',
```

Comment:
```suggestion
  title: 'components/features/ChartHeader',
```
는 어떨까요?! 현재 스토리북 코드 관련 컨벤션으로 `components/layer명/컴포넌트명` 으로 사용 중이에요!

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/89/commits/907065fce8510a4d8a5294a4b08e049c22218b71 에서 반영했습니다! 감사합니다ㅎㅎ